code/client/munkilib/munkicommon.py

```diff
@@ -1208,7 +1208,8 @@ def pref(pref_name):
         'SuppressUserNotification': False,
         'SuppressAutoInstall': False,
         'SuppressStopButtonOnInstall': False,
-        'PackageVerificationMode': 'hash'
+        'PackageVerificationMode': 'hash',
+        'AllowInsecureConnections': True
```

Comment:
+1

Once there is thorough documentation on setting up munki using SSL and appropriate configurations server side, this would be good to have.